tests/esm-transpile-only/index.ts

```diff
@@ -0,0 +1,12 @@
+import { foo } from './foo.js'
+import { bar } from './bar.js'
+import { baz } from './baz.js'
+import { biff } from './biff.js'
```

Comment:
Do you think we can remove the foo, bar, baz, and biff files?  Since `esm-transpile-only` is focused on verifying the transpile-only loader, so the focus is on the semantic errors.

Reply:
Sounds good; removed them.